osutil/disks/disks.go

```diff
@@ -187,3 +187,25 @@ func (e PartitionNotFoundError) Error() string {
 var (
 	_ = error(PartitionNotFoundError{})
 )
+
+var deviceMapperHandlers = map[string]func(dmUUID, dmName []byte) (dev, devType string, ok bool){}
+
+// RegisterDeviceMapperHandler takes a callback function which is used when the
```

Comment:
Handler is a bit vague to me,   BackResolver ?  as it resolve the backing storage of or something similar?

Reply:
BackResolver seems fine to me, renamed in https://github.com/snapcore/snapd/pull/11035